### ISSUE TITLE
`test_gp_kernel` error "fixture 'mocker' not found"

### ISSUE THREAD

**Issue by elcorto** [aleximmer/Laplace#214]
Running the tests for current `main` (508843dbc37498e02ef421a573e7c546a90a9443), I found this:

```
$ python -u -m pytest -k test_gp_kernel
[...]
  def test_gp_kernel(
E       fixture 'mocker' not found
>       available fixtures: ...
[...]
ERROR tests/test_functional_laplace_unit.py::test_gp_kernel[kernel_batch-jacobians0-jacobians_20-expected_full_kernel0-expected_block_diagonal_kernel0]
ERROR tests/test_functional_laplace_unit.py::test_gp_kernel[kernel_star-jacobians1-jacobians_21-expected_full_kernel1-expected_block_diagonal_kernel1]
ERROR tests/test_functional_laplace_unit.py::test_gp_kernel[kernel_batch_star-jacobians2-jacobians_22-expected_full_kernel2-expected_block_diagonal_kernel2]
```

**Comment by Ludvins** [comment]
This is my fault, I forgot to add pytest-mock package to the setup file. I can make a PR later today/tomorrow fixing this. To make the pytest work you only need to install pytest-mock via pip.

**Comment by elcorto** [comment]
Thanks & no rush :) I was wondering why the CI didn't choke on that, but it installs `pytest-mock`, so it only shows when running tests locally. In the CI, you could do `pip install -e ".[tests]"` once `pytest-mock` is in `setup.cfg`.

**Comment by wiseodd** [comment]
Fixed #215
